frontend/locale/en-US.json

```diff
@@ -0,0 +1,8 @@
+{
+  "product_list_add_to_cart": {
+    "modal": {
+      "message": "This product has options",
```

Comment:
This should have a better explanation.
This is what the is shown on the favoirtes lists when you click on the add to cart button for a configurable product. I think this add to cart button on product lists should look the same 
![image](https://user-images.githubusercontent.com/11764719/57721204-3ecfb280-7649-11e9-8819-19855a8b89e0.png)

Reply:
Updated.

Reply:
> Updated.

I think we should change the copy for the dismiss button too
![image](https://user-images.githubusercontent.com/11764719/57786656-8281f500-76f9-11e9-984c-c43654242153.png)
(sorry)

Reply:
Gotcha. Updated!